tests/gmsh_test.cc

```diff
@@ -0,0 +1,72 @@
+// GMSH test
+#include <array>
+#include <limits>
+
+#include "catch.hpp"
+#include "point.h"
+#include "gmsh.h"
+
+//! \brief Check that IO will store right values
+TEST_CASE("GMSH is checked in 3D", "[GMSH][3D]") {
+
+  //! 3D Test of IO Class
+  const unsigned DIM = 3;
+  const double tolerance = 1.E-12;
+
+  auto mesh = std::unique_ptr<Mesh<3, 8>> (new GMSH<3, 8>());
+  mesh->read_mesh("../bin/regression_test.msh");
```

Comment:
Could this be passed as an argument? rename `regression_test.msh` to a more meaningful filename of the geometry you are modelling like `cube_test.msh`

Reply:
I can change the name easily... as for the argument, I checked this: https://github.com/philsquared/Catch/blob/master/docs/own-main.md

Could you explain more on how to get this? I can't seem to add `int main` in `regression_test.cc` file